libs/feature/record/src/lib/map-view/map-view.component.html

```diff
@@ -1,13 +1,14 @@
 <div class="w-full h-full flex flex-col p-1">
-  <div class="w-full mb-7 mt-1">
+  <div class="w-full flex justify-end mb-7 mt-1">
     <gn-ui-dropdown-selector
+      class="truncate p-1 -mx-1"
```

Comment:
I'm surprised that this does anything; for me the dropdown button already has a truncate built-in if the width is too low for the text to show

Reply:
Yes, strangely this is necessary, because of the flex I added in the parent.